seed/python-sdk/auth-environment-variables/src/seed/client.py

```diff
@@ -26,6 +28,7 @@ class SeedAuthEnvironmentVariables:
     from seed.client import SeedAuthEnvironmentVariables
 
     client = SeedAuthEnvironmentVariables(
+        x_another_header="YOUR_X_ANOTHER_HEADER",
```

Comment:
@armandobelardo (for the future) we should do something smart in the IR Generation that removes `x_` for any headers

Reply:
(for the header name)

Reply:
Ah that's fair, I thought we did it now, but I think it's only for openAPI headers, not ones from Fern defs